docs/en/sql-reference/statements/grant.md

```diff
@@ -257,16 +327,17 @@ Hierarchy of privileges:
     - `S3`
     - `SQLITE`
     - `URL`
-- [dictGet](#dictget)
-- [displaySecretsInShowAndSelect](#displaysecretsinshowandselect)
-- [NAMED COLLECTION ADMIN](#named-collection-admin)
+- [`dictGet`](#dictget)
+- [`displaySecretsInShowAndSelect`](#displaysecretsinshowandselect)
+- [`NONE`](#none)
+- [`NAMED COLLECTION ADMIN`](#named-collection-admin)
     - `CREATE NAMED COLLECTION`
     - `DROP NAMED COLLECTION`
     - `ALTER NAMED COLLECTION`
     - `SHOW NAMED COLLECTIONS`
     - `SHOW NAMED COLLECTIONS SECRETS`
     - `NAMED COLLECTION`
-- [TABLE ENGINE](#table-engine)
+- [`TABLE ENGINE`](#table-engine)
 
```

Comment:
Here also should be the `SET DEFINER` privilege
https://clickhouse.com/docs/en/sql-reference/statements/create/view#sql_security